.github/workflows/image-push-master.yaml

```diff
@@ -0,0 +1,37 @@
+name: "push images on merge to master"
+
+env:
+  IMAGE_NAME: ghcr.io/${{ github.repository }}
+
+on:
+  push:
+    branches:
+      - master
+jobs:
+  build-and-push-ipoib-cni:
+    name: image push
+    runs-on: ubuntu-22.04
+    steps:
+      - name: check out the repo
+        uses: actions/checkout@v3
+
+      - name: set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: login to Docker
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: build and push ipoib-cni
+        uses: docker/build-push-action@v4
+        with:
+          context: .
+          push: true
+          platforms: linux/amd64,linux/arm64,linux/ppc64le
+          tags: |
+            ${{ env.IMAGE_NAME }}
+            ${{ env.IMAGE_NAME }}:${{ github.sha }}
+          file: ./Dockerfile
```

Comment:
so we dont use the Dockerfile.ppc64le ?

Reply:
personally if we dont need it, im OK with leaving multi-arch building only in github workflows

that is:
1. delete Dockerfile.ppc64le from project
2. remove the makefile target
3. remove mentions in README for ppc build.